index.html

```diff
@@ -872,6 +934,30 @@ <h3 class="bannerbox__title">Newsletter</h3>
 
             return hour + ':' + out_minutes + ampmString;
         }
+
+        function formatTime2(minutes) {
```

Comment:
Do we still need both these formatTime functions, or can we get rid of one of them? If we need both, maybe we should name them more descriptively.